### PR TITLE
Tablib and tabulate

### PR DESCRIPTION
**Pull request by cpennington**
Build commandline and csv files using the tablib and tabulate libraries, to offload the logic (and make manipulating the data easier).

### CODE REVIEW

**Comment by cpennington** [comment]
This depends on #258.

**Comment by fatso83** [comment]
This PR does not seem to have any blockers as it is. Is the project abandoned or in need of new maintainers? Nothing has been committed in four months.

**Comment by cgoldberg** [comment]
@fatso83 see #355

**Comment by cpennington** [comment]
@cgoldberg: I just rebased this, it should be merge-able now.

**Comment by mbeacom** [comment]
@cpennington Would you consider rebasing this to move forward with review?

File: setup.py [hunk 1]
```diff
@@ -40,7 +40,7 @@
     packages=find_packages(exclude=['ez_setup', 'examples', 'tests']),
     include_package_data=True,
     zip_safe=False,
-    install_requires=["gevent==1.1.1", "flask>=0.10.1", "requests>=2.9.1", "msgpack-python>=0.4.2", "six>=1.10.0", "pyzmq==15.2.0"],
+    install_requires=["gevent==1.1.1", "flask>=0.10.1", "requests>=2.9.1", "msgpack-python>=0.4.2", "six>=1.10.0", "pyzmq==15.2.0", "tablib", "tabulate"],
```
Comment (aldenpeterson-wf) [inline setup.py]: this should explicitly pin so that if something changes in those it doesn't break usage here

**Comment by cyberw** [comment]
Hi @cpennington ! This has been idle for a long time and now has conflicts, so I am closing it. Feel free to reopen if this is still important to you!

**Pull request closed without merging**